langserver/handle_text_document_formatting.go

```diff
@@ -77,6 +80,23 @@ func (h *langHandler) formatting(uri DocumentURI) ([]TextEdit, error) {
 		}
 		command = replaceCommandInputFilename(command, fname)
 
+		for placeholder, value := range options {
+			re, err := regexp.Compile(fmt.Sprintf(`\${([^:|^}]+):%s}`, placeholder))
+			if err != nil {
+				h.logger.Println(command+":", err)
+				continue Configs
+			}
+
+			switch v := value.(type) {
+			default:
+				command = re.ReplaceAllString(command, fmt.Sprintf("$1 %v", v))
+			case bool:
+				command = re.ReplaceAllString(command, "$1")
```

Comment:
this only supports `true`
I'm not sure how `false` should behave. Should it just be the same as not sending it?

Reply:
keep it so. thanks.